README.md

```diff
@@ -28,8 +28,22 @@ sudo apt install python3-pydbus
 
 ## Installation
 
-The extension is not yet signed by Mozilla and hence can only be added
-as temporary extension. For that, perform the following steps:
+### Firefox: via addons.mozilla.org
+
+You can get a signed version of the browser extension from AMO (the Mozilla addons marketplace).
+As this only covers the browser part, the host tooling still needs to be installed manually:
```

Comment:
Add a link to the marketplace? It's not yet online, though. Or didn't you even register?

Reply:
Done. It already got (automatically) reviewed and accepted.